src/app/asset-grid/asset-grid.component.ts

```diff
@@ -699,6 +705,7 @@ export class AssetGrid implements OnInit, OnDestroy {
 
   private saveReorder(): void {
     this.isLoading = true;
+    this.results = this.visibleResults
```

Comment:
As mentioned offline, I think this will remove any items that are no longer available because they were deleted by the creator. 

Currently we give a warning message to the user with the option to cull those deleted items (see image below) but reordering will do that now automatically. Is there any situation where an item might not be "available" but the user would still want to keep it in their group? Does availability only change when an item is removed or could it be updated when institutional access is removed? 

![image](https://user-images.githubusercontent.com/13315416/66336331-dd1b2800-e90a-11e9-8953-9da45677bc42.png)